### DISCUSSION TITLE
"There are no verified domains"

### DISCUSSION THREAD

**Discussion by lanlertpanit** [community/community#28276] [Other Feature Feedback, Questions, & Ideas]
Hello, 

This is my first time trying to publish my website here on GitHub. I followed the steps (adding the files to my new repos and such.) Then when I got the part Setting > Page, I see " There are no verified domain", and just like that, I got stuck :(
![Screen Shot 2022-08-03 at 6 42 46 PM](https://user-images.githubusercontent.com/110575138/182744014-55b6421a-57e9-4ac6-a9f5-2733304828b3.png)

Could someone please help me to solve this issue? I have not received a link from Github to verified my email btw.. would that be the issue? I tried to resend the link, but could not even re-send it.

Thank you for your time.

**Comment by Varefox** [comment]
Did you ever figure out the issue?

**Comment by riyazparve** [comment]
Go to your website repositry https://github.com/username/yourwebsite 
Click on settings
Remove your custom domain name and check Use your GitHub Pages website 

![image](https://github.com/community/community/assets/29349161/9581c6be-b5b0-4177-ae2d-23ff82f22dfd)
